docs/advanced.md

```diff
@@ -10,20 +10,19 @@ will use pgai to embed, moderate, and summarize git commit history.
 
 ## Load the sample data
 
-To add the advanced examples to your developer environment:
+To add the advanced examples to your developer environment, in the `<pgai-repo>/docs` folder:
```

Comment:
I don't get this change

Reply:
So the user knows they have to be in the basedir of this repo cloned on their local machine.